src/views/AboutUs.vue

```diff
@@ -38,8 +38,54 @@ v-container(fluid='' style='margin: 0px; padding: 0px; width: 100%;')
                   h1.font-weight-medium.pa-3(style="text-align: center;color:#000") {{item.heading}}
                   h3.font-weight-light.mb-n.pa-3(style="text-align: center;color:#000") {{item.subheading}}
               v-flex(xs12='',sm12='', md4='',lg4='',xl4='')
-                v-img.mt-5.ml-n3.mr-3(:src="item.src" contain aspect-ratio='2.5')    
+                v-img.mt-5.ml-n3.mr-3(:src="item.src" contain aspect-ratio='2.5')   
 
+
+  div(align='center', class="form-container" justify='center')
+    v-section.Form
+      div.container.col-sm-10(class="my-5")
+        v-row(class="no-gutters pb-5" id="formRow")
+          div.col-lg-5.col-md-5(class="pt-11 pl-7 pb-0" align="center" justify="center")
+            v-img.img-fluid(src="../assets/contactUs.png" id="contactImg" align="center" justify="center")
+          div.col-lg-7.col-md-7( justify='center' class='px-5')
+            div.col-lg-9(align="center" justify="left")
+              iframe(name="hiddenFrame" width="0" height="0" border="0" style="display: none;"
+                    onload="if(submitted){location.reload()}")
+              v-form(class="login my-3" id="user_info_submit" name="form"
+                    action="https://docs.google.com/forms/u/1/d/e/1FAIpQLSd25Z-mQHDN42vNZdMXwQXkH_LfTnpUMA2kxJPnj4VHy6q8JQ/formResponse"
+                    method="POST" target="hiddenFrame"
+                    v-model="validForm"
+                    onsubmit="submitted=true;"
+                    )
+                h1(style="color: #1b65c4; font-size: 2.5rem" class="font-weight-bold py-3") Contact Us
+                div.form-row
+                    v-text-field(
+                                  lable="Email: " id="email" name="entry.1624474703" color='#1b65c4' 
+                                  type="email"  placeholder="Email" 
+                                  v-model="emailAddress"
+                                  :rules="emailRules")
+                div.form-row
+                    v-text-field(
+                                  lable="full name: " id="name" name="entry.274700485" color='#1b65c4' 
+                                  type="text" placeholder="Full Name" 
+                                  counter=50
+                                  :rules="[required('Name'), minLength('Name', 4), maxLength('Name', 50)]")
+                div.form-row
+                    v-text-field(
+                                  lable="phone" color='#1b65c4' id="phone" name="entry.1452831997" 
+                                  type="phone" placeholder="Phone"
```

Comment:
Change the name Phone to Contact Number and use Contact Number as the placeholder